README.md

```diff
@@ -80,15 +80,98 @@ See below for more details.
+
+It is often helpful to view the log output in another terminal window while the test suite is running:
+
+```bash
+# Follow the log file
+$ tail -f $PANDIR/logs/panoptes.log
+```
+
+#### Testing your installation
+
+In order to test your installation you should have followed all of the steps above for getting your unit ready. To run the test suite, you will need to open a terminal and navigate to the `$POCS` directory.
+
+```bash
+# Change to $POCS directory
+(panoptes-env) $ cd $POCS
+
+# Run the software testing
+(panoptes-env) $ pytest
+```
+
+The output from this will look something like:
 
 ```bash
-cd ${POCS}
-pytest
+(panoptes-env) $  pytest                                                                                                                                                     
+=========================== test session starts ======================================
+platform linux -- Python 3.5.2, pytest-3.2.3, py-1.4.34, pluggy-0.4.0                                                 
+rootdir: /storage/panoptes/POCS, inifile:                       
+plugins: cov-2.4.0                                                                                                     
+
+collected 260 items                                                                                                                                                                                                                   
+pocs/tests/test_base_scheduler.py ...............
+pocs/tests/test_camera.py ........s..ssssss..................ssssssssssssssssssssssssss
+pocs/tests/test_codestyle.py .
+pocs/tests/test_config.py .............
+pocs/tests/test_constraints.py ..............
+pocs/tests/test_database.py ...
+pocs/tests/test_dispatch_scheduler.py ........
+pocs/tests/test_field.py ....
+pocs/tests/test_focuser.py .......sssssss..
+pocs/tests/test_images.py ..........
+pocs/tests/test_ioptron.py .
+pocs/tests/test_messaging.py ....
+pocs/tests/test_mount_simulator.py ..............
+pocs/tests/test_observation.py .................
+pocs/tests/test_observatory.py ................s.......
+pocs/tests/test_pocs.py ..........................
+pocs/tests/test_utils.py .............
+pocs/tests/bisque/test_dome.py ssss
+pocs/tests/bisque/test_mount.py sssssssssss
+pocs/tests/bisque/test_run.py s
+
+=========================== 203 passed, 57 skipped, 6 warnings in 435.76 seconds ===================================
+
 ```
 
-This may take 5 to 10 minutes as there are a lot of tests to run! If you experience any errors, ask for check the [Issues](https://github.com/panoptes/POCS/issues) listed above or ask one of our friendly team members!
+Here you can see that certain tests were skipped (`s`) for various reasons while the others passed. Skipped tests are skipped on purpose and thus are not considered failures. Usually tests are skipped because there is no attached hardware (see below for running tests with hardware attached).
```

Comment:
Perhaps some more detail on how to read the output: a . for each passed test, an f for each failed test, etc?